mne/decoding/transformer.py

```diff
@@ -245,6 +245,111 @@ def inverse_transform(self, X, y=None):
         return X.reshape(-1, self.n_channels, self.n_times)
 
 
+class Vectorizer(TransformerMixin):
```

Comment:
@agramfort, I added a deprecate warning and replaced `EpochsVectorizer` with `Vectorizer` in the examples it was being used.